card/spec/models/card/trash_spec.rb

```diff
@@ -118,7 +118,7 @@
       @a.delete!  #trash
       Rails.logger.info "\n\n~~~~~~~deleted~~~~~~~~\n\n\n"
 
-      @b.update_attributes! name: "A", update_referencers: true
+      @b.update_attributes! name: "A", update_referers: true
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.